### PR TITLE
Export route and connect from addon root.

### PR DESCRIPTION
**Pull request by brettburley**
This is an example implementation for #112.

### CODE REVIEW

File: addon/connect.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import Ember from 'ember';
```
Comment (brettburley) [inline addon/connect.js]: This is just a move (no changes) from its original location.

**Comment by toranb** [comment]
@brettburley can't say I have a problem with this - seems to make sense. Anyone else pros/cons to making this change (aside from the find/replace for import statements when we land a v3.0.0 later this year) ?

**Comment by toranb** [comment]
@brettburley I went ahead and rebased this branch - it seems g2g but I wanted your final input before we roll ahead. On a related note, this made the typescript definition work much easier so that is a large reason I'm suddenly motivated to land this ;)

**Comment by brettburley** [comment]
Yep, this still looks good to me!

**Comment by toranb** [comment]
@brettburley this is now published in ember-redux v2.7.0 (I also updated the guides to reflect this new import location)

**Pull request merged**